src/app/containers/ConsentBanner/index.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { Fragment } from 'react';
+import ConsentBanner from '../../components/ConsentBanner';
+import { PlatformContextConsumer } from '../../contexts/PlatformContext';
+import { ServiceContextConsumer } from '../../contexts/ServiceContext';
+
+const parentId = 'consent';
+const promptId = 'consent-prompt';
+
+const ampOnTapHandler = type => ({ on: `tap:${parentId}.${type}` });
+
+const ampGeoData = {
+  ISOCountryGroups: {
+    eu: [
+      'at',
+      'be',
+      'bg',
+      'cy',
+      'cz',
+      'de',
+      'dk',
+      'ee',
+      'es',
+      'fi',
+      'fr',
+      'gb',
+      'gr',
+      'hr',
+      'hu',
+      'ie',
+      'is',
+      'it',
+      'li',
+      'lt',
+      'lu',
+      'lv',
+      'mt',
+      'nl',
+      'no',
+      'pl',
+      'pt',
+      'ro',
+      'se',
+      'si',
+      'sl',
+    ],
+  },
+};
+
+const ampConsentData = {
+  consents: {
+    'user-consent': {
+      promptIfUnknownForGeoGroup: 'eu',
+      promptUI: promptId,
+    },
+  },
+};
+
+const jsonInlinedScript = data => (
+  <script
+    type="application/json"
+    /* eslint-disable-next-line react/no-danger */
+    dangerouslySetInnerHTML={{ __html: JSON.stringify(data) }}
+  />
+);
+
+const ConsentBannerContainer = () => (
+  <PlatformContextConsumer>
+    {platform => {
+      if (platform === 'amp') {
+        return (
+          <Fragment>
+            <amp-geo layout="nodisplay">
```

Comment:
Im almost tempted to say this should go on psammead, should we make an issue to extract it as a component?

Reply:
https://github.com/bbc/psammead/issues/298 Done